tools/helm-docs/readme.gotmpl

```diff
@@ -51,6 +51,16 @@ To uninstall the chart:
     helm delete eg
 ```
 
+### Certgen
+
+Certgen is used to generate the certificates required by EnvoyGateway.
```

Comment:
this should live in https://github.com/envoyproxy/gateway/blob/33fceb05ca611e30107ff841cac9d7762eea2048/charts/gateway-helm/values.tmpl.yaml#L68 which then gets generated using `make helm-docs`

Reply:
OK, I added it in `values.tmpl.yaml`.

At the same time, I kept the description in `readme.gotmpl`: I think certgen Job determines whether some functions of eg can be used normally, so I hope to inform users of its importance through "**multiple emphasis**".

Reply:
Can we move all the doc strings into `values.tmpl.yaml` where the doc string is meant to live, the `.gotmpl` is only meant for editing presentation

Reply:
Sorry, I misunderstood. We should indeed move this description text to `values.tmpl.yaml`, I will change it!